documentation/monitoring.md

```diff
@@ -75,6 +75,44 @@ There are several other variables that can be changed depending on env requireme
 - grafana_app_mem - app memory limit (default 1Gi)
 - grafana_app_cpu - app requests cpu (default 500m)
 
+Grafana could be accessed via the respective URL's based on the environment of interest. The URL's corresponding to each environment is as below
```

Comment:
This card is "Document monitoring URLs for devs"
This doc here is not for developers, it's infra team documentation. Help for devs should go to the [developer-onboarding.md](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/developer-onboarding.md) doc. Please move all the content there.

Reply:
moved